### PR TITLE
feat(trends): added persons modal to all trends graph types

### PR DESCRIPTION
**Pull request by Gilbert09**
## Problem
- We added the new persons modal with the insight actor query to the line chart on trends, but had forgotten to add this to all the other chart types (e.g. pie chart, bold number, bar chart)

## Changes
- Adds the person modal to the bold number chart, pie chart, and bar chart

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?
- Clicking around in the browser

### CODE REVIEW

**Review by mariusandra — COMMENTED** [review]
Nice! Some quirks though:

- Found this for the pie chart:
image

- The numbers don't look right here:


The same case occurs with line charts

**Comment by Gilbert09** [comment]
> Nice! Some quirks though:
> 
> * Found this for the pie chart:
> 
> image
> * The numbers don't look right here:
>   
>         
>           [
>             
>           
>               
>             ](https://private-user-images.githubusercontent.com/53387/297688832-70b21710-7a53-42bc-b820-7f7ab25ccd61.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU1OTEwMzQsIm5iZiI6MTcwNTU5MDczNCwicGF0aCI6Ii81MzM4Ny8yOTc2ODg4MzItNzBiMjE3MTAtN2E1My00MmJjLWI4MjAtN2Y3YWIyNWNjZDYxLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE4VDE1MTIxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU4MzcxOWMyMTFhMWVjMTdhNDgwMmYxNWQ1YmI0MjRhZDkzOGZhOTJmNjdiYzE5MmUzNjE5NTZkOGFmYjc2MzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.yBaMFT9NjArjyxjLcNT7DTIAJJrXN8FVqkboUJrlXqw)
>           
>           
>             
>               
>                 
>               
>               
>                 
>                 
>               
>             
>             [
>               
>                 
>               
>             ](https://private-user-images.githubusercontent.com/53387/297688832-70b21710-7a53-42bc-b820-7f7ab25ccd61.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU1OTEwMzQsIm5iZiI6MTcwNTU5MDczNCwicGF0aCI6Ii81MzM4Ny8yOTc2ODg4MzItNzBiMjE3MTAtN2E1My00MmJjLWI4MjAtN2Y3YWIyNWNjZDYxLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE4VDE1MTIxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU4MzcxOWMyMTFhMWVjMTdhNDgwMmYxNWQ1YmI0MjRhZDkzOGZhOTJmNjdiYzE5MmUzNjE5NTZkOGFmYjc2MzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.yBaMFT9NjArjyxjLcNT7DTIAJJrXN8FVqkboUJrlXqw)
> 
> The same case occurs with line charts

@mariusandra All fixed. Trends returns the correct values now (no support for breakdowns though, it'll just show the "total count" for the breakdowns combined on that period), and the persons modal handles the "Other" value correctly

File: frontend/src/scenes/trends/persons-modal/PersonsModal.tsx [hunk 1]
```diff
@@ -85,6 +86,18 @@ export function PersonsModal({
...
     const totalActorsCount = missingActorsCount + actors.length
 
+    const getTitle = useCallback(() => {
+        if (typeof title === 'function') {
+            return title(capitalizeFirstLetter(actorLabel.plural))
+        }
+
+        if (title === BREAKDOWN_OTHER_STRING_LABEL) {
+            return 'Other'
+        }
```
Comment (mariusandra) [inline frontend/src/scenes/trends/persons-modal/PersonsModal.tsx:96]: What if it's the `BREAKDOWN_OTHER_NUMBER_LABEL` ? We have somewhere a `isOtherBreakdown` function that'd be perfect here.

Personally I wouldn't have bothered with a `useCallback` here, and just inlined this into the body of the component. I doubt there are any measurable performance wins here. However to be truly accurate, you'd need to add `actorLabel.plural` also to the deps array.

**Review by mariusandra — APPROVED** [review]
I love forward progress. I left one comment inline that should be addressed (breakdown util function), and one that I'm somewhat indifferent to (useCallback). Here's a ✅ to unblock though.

**Pull request merged**